Alignment/MuonAlignmentAlgorithms/plugins/MuonMillepedeTrackRefitter.h

```diff
@@ -13,9 +13,15 @@
 #define Alignment_MuonMillepedeTrackRefitter_MuonMillepedeTrackRefitter_H
 
 // Base Class Headers
-#include "FWCore/Framework/interface/EDProducer.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
```

Comment:
Could you please confirm if the private test is done with multicore, and you still get the memory in control, i.e. this module can run on multiple streams.

Reply:
I've tried to validate and run on multiple streams but we never use this module recently and the cosmic dataset is not working with this module. 
I think we can clean up the millipede method in the muon alignment modules.
However, is it fine?

Reply:
@hyunyong I'm not sure I understand "we can clean up the millipede method in the muon alignment modules.", can you please clarify? 
Also "the cosmic dataset is not working with this module", couldn't you just use some older data for the tests?

Reply:
@tvami We don't have a plan to use this millipede method anymore. So, I think we can remove the module but I don't know the CMSSW policy.

Reply:
@srimanob , @mmusich what do you think?

Reply:
If the class is not used it can just be deleted

Reply:
Great! Then @hyunyong please go ahead with the removal

Reply:
@tvami Yes, I've removed it.